tests/integration/routes/accounts/delete-accounts-test.js

```diff
@@ -91,8 +91,21 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not an admin', {todo: true}, function (t) {
-      t.end()
+    group.test('Not an admin', function (t) {
+      server.inject({
+        method: 'DELETE',
+        url: '/accounts/abc4567',
+        headers: {
+          accept: 'application/vnd.api+json',
+          authorization: 'Session cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ',
+          'content-type': 'application/vnd.api+json'
```

Comment:
you can remove the `content-type`, it’s only needed when you send a payload (usually `PUT`, `PATCH` and `POST` requests only)